packages/services/policy/package.json

```diff
@@ -9,17 +9,18 @@
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
-    "@graphql-eslint/eslint-plugin": "3.20.1",
+    "@graphql-eslint/eslint-plugin": "4.4.0",
     "@hive/service-common": "workspace:*",
     "@sentry/node": "7.120.2",
     "@sentry/tracing": "7.114.0",
     "@trpc/server": "10.45.2",
-    "@types/eslint": "8.56.12",
+    "@types/eslint": "9.6.1",
     "ajv": "8.17.1",
     "dotenv": "16.4.7",
-    "eslint": "8.57.1",
+    "eslint": "9.23.0",
     "fastify": "4.29.0",
     "graphql": "16.9.0",
+    "json-schema-to-ts": "3.1.1",
```

Comment:
what is this dependency used for?

Reply:
It added it because `graphql-eslint` uses it internally to create the programmatic API types for the rules config schema. Without it, it was all `any`, but now the test samples are also fully type-checked.